content/docs/06-integrations/00-frp.md

```diff
@@ -16,39 +16,331 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
 
 # Spectro Proxy
-Spectro Proxy is a fast and straightforward reverse proxy that lets you forward a port of your local server behind a NAT or firewall to a public server. The proxy server pack is available as an add on pack for authentication. Users can attach this layer to the cluster profile while profile creation. This installs the FRP client in the workload clusters and configures it with a FRP server. Spectro Cloud provides hosts FRP server and by default the pack is configrued to connect to this server. Spectro Cloud detects the presence of this pack in the cluster and automatically updates the Kubeconfig file to use the FRP server as the endpoint.
+Spectro Proxy is a pack that enables the usage of a reverse proxy with a Kubernetes cluster. The reverse proxy allows you to connect to a Palette-managed Kubernetes cluster's API in private networks or Kubernetes clusters configured with private API endpoints. The reverse proxy managed by Spectro Cloud is also known as the forward reverse proxy (FRP).
+
+The reverse proxy comprises two components, a server, and a client. The reverse proxy server is publicly available and managed by us. The client runs inside your Palette-managed Kubernetes cluster and connects to the reverse proxy server. When you add the Spectro Proxy pack to a cluster profile, a couple of things happen.
+
+<br />
+
+- The KubeConfig file is updated with the reverse proxy address instead of pointing directly to the Kubernetes cluster's API address. The following is an example of a kubeconfig file where the `server` attribute points to the reverse.
+
+    ```yaml
+    apiVersion: v1
+    clusters
+    - cluster:
+    certificate-authority-dataLSOtLa....
+    server: https: //cluster-11111111111111.proxy.stage.spectrocloud.com:443
+    name: vsphere-proxy
+    contexts:
+    - context:
+    cluster: vsphere-proxy
+
+    ```
+
+- Any requests to the Kubernetes API server, such as kubectl commands, will be routed to the reverse proxy. The reverse proxy will forward the request to the intended client, which is the Kubernetes cluster's API server. The Kubernetes cluster's API server will authenticate the request and reply with the proper response.
+
+
+Users can attach this pack to a [cluster profile](/cluster-profiles). The pack installs the Spectro Proxy client in the workload clusters and configures the cluster's API server to point to a managed proxy server.
+
+<br />
+
 
 <InfoBox>
 
-Port 443 needs to be Open for Outbound Communication. <br />
-Do not change any values which are available by default, as it is required by our backend system to set the correct configurations.
 
+This pack can be combined with the [Kubernetes dashboard](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/) pack to expose the Kubernetes dashboard. To learn more about exposing the Kubernetes dashboard, check out the [Enable Kubernetes Dashboard](/clusters/cluster-management/reverse-proxy-dashboard) guide. 
```

Comment:
The "Enable Kubernetes Dashboard" link doesn't work. Is it because **https://** is missing in the path?